Converters/Autocad/Speckle.Converters.AutocadShared/ToHost/Raw/RegionHatchToHostRawConverter.cs

```diff
@@ -0,0 +1,153 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Objects;
+using Speckle.Sdk;
+using Speckle.Sdk.Common.Exceptions;
+using Speckle.Sdk.Models;
+
+namespace Speckle.Converters.Autocad.ToHost.Raw;
+
+public class RegionHatchToHostRawConverter : ITypedConverter<SOG.Region, ADB.Hatch>
+{
+  private readonly ITypedConverter<ICurve, List<(ADB.Entity, Base)>> _curveConverter;
+  private readonly IConverterSettingsStore<AutocadConversionSettings> _settingsStore;
+
+  public RegionHatchToHostRawConverter(
+    ITypedConverter<ICurve, List<(ADB.Entity, Base)>> curveConverter,
+    IConverterSettingsStore<AutocadConversionSettings> settingsStore
+  )
+  {
+    _curveConverter = curveConverter;
+    _settingsStore = settingsStore;
+  }
+
+  public ADB.Hatch Convert(SOG.Region target)
+  {
+    // Get the current document and database
+    Document acDoc = _settingsStore.Current.Document;
+    ADB.Database acCurDb = acDoc.Database;
+
+    // Start a transaction
+    ADB.Transaction acTrans = acCurDb.TransactionManager.StartTransaction();
+    // Open the Block table for read
+    ADB.BlockTable? acBlkTbl = acTrans.GetObject(acCurDb.BlockTableId, ADB.OpenMode.ForRead) as ADB.BlockTable;
+    if (acBlkTbl == null)
+    {
+      throw new ConversionException();
+    }
+
+    // Open the Block table record Model space for write
+    ADB.BlockTableRecord? acBlkTblRec =
+      acTrans.GetObject(acBlkTbl[ADB.BlockTableRecord.ModelSpace], ADB.OpenMode.ForWrite) as ADB.BlockTableRecord;
+    if (acBlkTblRec == null)
+    {
+      throw new ConversionException();
+    }
+
+    // initialize Hatch, only once, with the boundary
+    ADB.Hatch acHatch = InitializeHatchObject(acBlkTblRec, acTrans);
+
+    // convert and assign boundary loop
+    // catch any exception to finish the transaction, throw it later
+    Exception? exception = null;
+    try
+    {
+      ConvertAndAssignHatchLoop(acBlkTblRec, acTrans, acHatch, target.boundary, ADB.HatchLoopTypes.External);
+      foreach (var loop in target.innerLoops)
+      {
+        ConvertAndAssignHatchLoop(acBlkTblRec, acTrans, acHatch, loop, ADB.HatchLoopTypes.Outermost);
+      }
+    }
+    catch (Exception ex) when (!ex.IsFatal())
```

Comment:
If you're using the top level transaction, you don't need to commit it within the conversion block.
Not sure why you are catching the exception here, only to rethrow later? If you catch the exception, you should be handling it in some way (returning null hatch eg) instead of just rethrowing it later